### DISCUSSION TITLE
How to improve pgroonga performance at common mechanical disks ?

### DISCUSSION THREAD

**Discussion by Sanbomsu** [pgroonga/pgroonga#412] [Q&A]
How to improve pgroonga performance at common mechanical disks ? 

When I inserted data into database at a rate of about 100 records per second , sometimes the target table was locked at common mechanical disks (never happened at SSD). I have to rename the table, reindex the table and clear the lock for recovery, so it is inefficient. Is there a way to solve this problem ? 

### Environment
`pgroonga verison 2.2.9`

**Comment by kou** [comment]
What does "lock" mean? Could you explain why you think that "lock" is the problem?
